Flatiron - Tuples chapter.md

```diff
@@ -0,0 +1,115 @@
+
+##What's a Tuple?
+Tuples are a very specialized way of grouping and organizing data, and to best understand their value, limitations and purpose we'll compare and contrast them to Arrays, Dictionaries and Classes.
+
+**Arrays** help you manage variably-sized ordered collections of similar data, such as a list of numbers or strings.  For example, we can make an array of some key characters from the movie Star Wars:
+```
+// Array
+let list = [ "Darth", "Ben", "Luke" ]
+print( list[1] )
+```
```

Comment:
When creating code snippets in Swift through Markdown, we like to include the word swift in the snippet like so so it formats correctly.  
![swift snippet](http://i.imgur.com/b8FVZG7.png)
.. turns into this
```swift
let name = "Jim"
```

In applying that logic to the code you wrote above, it would look like this instead:

```swift
let list = ["Darth", "Ben", "Luke"]

print(list[1])
// prints "Ben"
```